### PR TITLE
Prepare to 0.21.0

### PR DESCRIPTION
**Pull request by Bugagazavr**
Prepare documentation to 0.21.0 release, it's just a README and version, no release

### CODE REVIEW

**Comment by dblock** [comment]
@Bugagazavr When you make a release, just commit these changes, and make the release, no need for PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +10,7 @@
 * [#416](https://github.com/ruby-grape/grape-swagger/pull/416): Support recursive models - [@lest](https://github.com/lest).
 * [#419](https://github.com/ruby-grape/grape-swagger/pull/419): Replaced github ref to rubygems for external gems - [@Bugagazavr](https://github.com/Bugagazavr).
 * [#420](https://github.com/ruby-grape/grape-swagger/pull/420): Raise SwaggerSpec exception if swagger spec isn't satisfied, when no parser for model is registered or response model is empty - [@Bugagazavr](https://github.com/Bugagazavr).
+* Your contribution here.
```
Comment (stefan-kolb) [inline CHANGELOG.md]: You may remove this line.

**Comment by LeFnord** [comment]
Please make a rebase, so actual fixes are included, thanks

**Comment by Lordnibbler** [comment]
Can we get this released, and include https://github.com/ruby-grape/grape-swagger/pull/441?

**Comment by Bugagazavr** [comment]
if we have no critical bugs, ithink we can release that, what you think @LeFnord ?

**Comment by LeFnord** [comment]
@Bugagazavr one last question: did you think a user can easy use the new version,
or should we add a guide on how to upgrade from 0.20.x → 0.21.0?
if you think it's fine, then you have my 👍 

and thanks for your work 😄 …

**Comment by LeFnord** [comment]
@Lordnibbler … it would be released from master, so it should be include all actual work including changelog, or if not @Bugagazavr rebase again please

**Comment by Bugagazavr** [comment]
@LeFnord yes i think this can be good to release a new major version.

**Comment by Lordnibbler** [comment]
\o/

**Comment by dblock** [comment]
If there's something to be done manually during an upgrade from 0.20 -> 0.21, please make sure to explain it in UPGRADING.

**Comment by Bugagazavr** [comment]
0.21.0 published

**Pull request closed without merging**